glib2: prefix glib binaries in pkgconfig file with BIN_DIR

Make sure the host versions of the glib binaries are used.
---
 glib-2.0.pc.in |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

Index: glib-2.18.4/glib-2.0.pc.in
===================================================================
--- glib-2.18.4.orig/glib-2.0.pc.in
+++ glib-2.18.4/glib-2.0.pc.in
@@ -1,11 +1,12 @@
+BIN_DIR=
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
 
-glib_genmarshal=glib-genmarshal
-gobject_query=gobject-query
-glib_mkenums=glib-mkenums
+glib_genmarshal=${BIN_DIR}glib-genmarshal
+gobject_query=$(BIN_DIR}gobject-query
+glib_mkenums=${BIN_DIR}glib-mkenums
 
 Name: GLib
 Description: C Utility Library
